CoinEx.Net/Clients/SpotApi/CoinExSocketClientSpotApi.cs

```diff
@@ -187,7 +187,7 @@ public async Task<CallResult<UpdateSubscription>> SubscribeToOrderBookUpdatesAsy
             mergeDepth.ValidateIntBetween(nameof(mergeDepth), 0, 8);
             limit.ValidateIntValues(nameof(limit), 5, 10, 20);
 
-            var subscription = new CoinExDepthSubscription(_logger, symbol, new object[] { symbol, limit, CoinExHelpers.MergeDepthIntToString(mergeDepth), false }, onMessage);
+            var subscription = new CoinExDepthSubscription(_logger, symbol, new object[] { symbol, limit, CoinExHelpers.MergeDepthIntToString(mergeDepth), true }, onMessage);
```

Comment:
@JanKorf It changes from getting the full book every time to diff updates

Reply:
Then this would change the functionality of the the function and break implementations relying on it sending the full updates without clear indication that it's changed. I'd rather add the boolean as an optional parameter which defaults to the old value.

Just as a heads up, CoinEx recently released a new version V2 of the API which I'm currently implementing so that will replace the current V1 api.

Reply:
I agree. I changed `diffUpdates` to be an optional argument. It had to be added after the Handler as it's optional